class.jetpack-gutenberg.php

```diff
@@ -497,7 +497,7 @@ public static function load_scripts_as_required( $type, $dependencies = array()
 			: array();
 		$script_dependencies = array_merge( $script_dependencies, $dependencies, array( 'wp-polyfill' ) );
 
-		if ( self::block_has_asset( $script_relative_path ) ) {
+		if ( ! Jetpack_AMP_Support::is_amp_request() && self::block_has_asset( $script_relative_path ) ) {
```

Comment:
Could you add a `class_exists( 'Jetpack_AMP_Support' )` class check here, since the class does not exist on WordPress.com yet?

Reply:
Resolved in https://github.com/Automattic/jetpack/pull/11318/commits/550035495b9d8cf54b8ac417e107b6b8278b978d